builder/virtualbox/common/driver_4_2.go

```diff
@@ -189,7 +186,18 @@ func (d *VBox42Driver) SuppressMessages() error {
 }
 
 func (d *VBox42Driver) VBoxManage(args ...string) error {
-	_, err := d.VBoxManageWithOutput(args...)
+	ctx := context.TODO()
+	err := retry.Config{
+		Tries: 5,
+		ShouldRetry: func(err error) bool {
+			return isLockedBySession(err.Error())
```

Comment:
Nit: Seeing as the isLockedBySession function is a one liner and not used anywhere else I would recommend replacing this line with then contents of the function; adding a comment mentioning that this error represents the VM being in a locked state.

```suggestion
			return strings.Contains(err.Error(), "VBOX_E_INVALID_OBJECT_STATE")
```

Reply:
thanks!! first I was thinking about making clear that is a session lock thing because the code is not that clear, but is a state thing as well so the method is not that necessary!